src/index.ts

```diff
@@ -78,4 +80,11 @@ export default {
     }
     return router.handle(request, env);
   },
+
+  //keeping these code under comment until we patch the pagination
+  //of overdue tasks API
+
+  // async scheduled(req: Request, env: env, ctx: ExecutionContext) {
+  //   ctx.waitUntil(send(env));
+  // },
```

Comment:
??
Kindly remove

Reply:
this will be uncommented once we fix the overdue task api, I have kept this here for testing on local and staging